.gitignore

```diff
@@ -1,4 +1,5 @@
 /.idea/
+/nbproject/
```

Comment:
if this folder is not directly related to the project it should be part of the global .`gitignore`.
I know that in theory the same would apply to `.idea/` but it's the first time I've seen it inside `Codeception/*`

Reply:
OK removed.